_pages/Ppdm/Parallel-place-of-manufacture.md

```diff
@@ -13,18 +13,18 @@ sidebar:
 1. TOC
 {:toc}
 
-## 4.212.1 Element information
+## 4.222.1 Element information
 
-<a name="4.212.1.1">4.212.1.1</a> [Link to RDA Toolkit](https://beta.rdatoolkit.org/Content/Index?externalId=en-US_ala-39b430be-31ff-3456-bf49-d1fdf7c823a0){:target="_blank"}
+<a name="4.222.1.1">4.222.1.1</a> [Link to RDA Toolkit](https://beta.rdatoolkit.org/Content/Index?externalId=en-US_ala-39b430be-31ff-3456-bf49-d1fdf7c823a0){:target="_blank"}
 
-## 4.212.2 RDA Definition and scope
+## 4.222.2 RDA Definition and scope
 
-<a name="4.212.2.1">4.212.2.1</a> A place of manufacture in another language or script.
+<a name="4.222.2.1">4.222.2.1</a> A place of manufacture in another language or script.
 
-## 4.212.3 Manufacture place names in more than one language or script
+## 4.222.3 Manufacture place names in more than one language or script
 
-<a name="4.212.3.1">4.212.3.1</a> If the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) appears in more than one language or script, transcribe the place in the language or script of the title proper as a [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) element.  If this criterion does not apply, transcribe the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) that appears first.
+<a name="4.222.3.1">4.222.3.1</a> If the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) appears in more than one language or script, transcribe the place in the language or script of the title proper as a [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) element.  If this criterion does not apply, transcribe the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) that appears first.
 
-<a name="4.212.3.2">4.212.3.2</a> Transcribe remaining [Places of manufacture](/DCRMR/ppdm/Place-of-manufacture/) as parallel places of manufacture. If a parallel place of manufacture is from a different source than the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/), make a [Note on manufacture statement](/DCRMR/ppdm/Note-on-manufacture-statement/) if considered important for identification (see [Note on manufacture statement](/DCRMR/ppdm/Note-on-manufacture-statement/), [4.226.7.1](/DCRMR/ppdm/Note-on-manufacture-statement/#4.226.7.1)).
+<a name="4.222.3.2">4.222.3.2</a> Transcribe remaining [Places of manufacture](/DCRMR/ppdm/Place-of-manufacture/) as parallel places of manufacture. If a parallel place of manufacture is from a different source than the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/), make a [Note on manufacture statement](/DCRMR/ppdm/Note-on-manufacture-statement/) if considered important for identification (see [Note on manufacture statement](/DCRMR/ppdm/Note-on-manufacture-statement/), [4.226.7.1](/DCRMR/ppdm/Note-on-manufacture-statement/#4.226.7.1)).
```

Comment:
Transcribe remaining Places of Publication" not hyperlinked in Parallel place of publication. This should be consistent.